Makefile

```diff
@@ -15,10 +15,8 @@ SOURCE_DIRS = cmd pkg test
 RELEASE_DIR ?= release
 
 # Docs build related variables
-DOCS_BUILD_DIR ?= docs/build
-DOCS_BUILD_CONTAINER ?= registry.gitlab.com/gbraad/asciidoctor-centos:latest
-DOCS_PDF_CONTAINER ?= quay.io/crcont/asciidoctor-pdf:latest
-DOCS_SERVE_CONTAINER ?= quay.io/crcont/docs-server:latest
+DOCS_BUILD_DIR ?= /docs/build
+DOCS_BUILD_CONTAINER ?= quay.io/anjan/docs-builder
```

Comment:
Need to change this to `quay.io/crcont/docs-builder` before merging, haven't pushed the image to crcont org yet!

Reply:
:+1: